releasenotes/notes/fix-quantum-kernel-duplicates-b75d6ed8a0f37f60.yaml

```diff
@@ -0,0 +1,23 @@
+---
+features:
+  - |
+    Introduced a new parameter `evaluate_duplicates` in
+    :class:`~qiskit_machine_learning.kernels.QuantumKernel`. This parameter defines a strategy how
+    kernel matrix elements are evaluated if identical samples are found.
+    Possible values are:
+
+      - ``all`` means that all kernel matrix elements are evaluated, even the diagonal ones when
+          training. This may introduce additional noise in the matrix.
+      - ``non_diagonal`` when training the matrix diagonal is set to `1`, the rest elements are
+          fully evaluated, e.g., for two identical samples in the dataset. When inferring, all
+          elements are evaluated. This is the default value.
+      - ``none`` when training the diagonal is set to `1` and if two identical samples are found
+          in the dataset the corresponding matrix element is set to `1`. When inferring, matrix
+          elements for identical samples are set to `1`.
+fixes:
+  - |
+    Fixed quantum kernel evaluation when duplicate samples are found in the dataset. Originally,
```

Comment:
`when duplicate samples` above, apart from the parameter `evaluate_duplicates` it uses `identical` to describe samples, Here it uses `duplicate` like the parameter does, but this is the only place. I guess I should read `duplicate` and `identical` as meaning the same thing - i.e. two or more samples having an identical value and thus being in essence duplicates of one another.

Reply:
That's right. I used both terms in the same meaning. I agree, I was not consistent. In terms of language and docstrings/parameters what is better?

Reply:
I didn't find it confusing to have 2 terms, I think it's pretty clear that they refer to the same concept. I believe that maybe balancing them out and using duplicate a bit more should be enough (see suggestions below).

Reply:
Yes, I would prefer to see duplicate used more in the descriptions when referring to elements managed by the evaluate_duplicates. 

I can see a duplicate as being another sample with an identical value as I mentioned above, but that begs another question I guess....
....when something is a duplicate is this an exact value match? - we have no tolerance here that can be set for when things are very very close that we consider them the same (a duplicate)?

Reply:
Replaced with `duplicated`, thanks to Elena